Update term import

zhengj2007 opened the pull request:
Solve issues related to term imported from external resources
resolves #1141 deprecated GO term
resolves #1135 not registered REO terms

jamesaoverton commented:
Thanks @zhengj2007. I noticed that REO:0000280 was still in `obi-edit.owl`, so I replaced it with OBI:0002744. I also added REO:0000171 and REO:0000280 to `obsolete.tsv` with their replacements noted, since OBI users may have used those terms.

Can you please double-check this for me?

zhengj2007 commented:
> Thanks @zhengj2007. I noticed that REO:0000280 was still in `obi-edit.owl`, so I replaced it with OBI:0002744. I also added REO:0000171 and REO:0000280 to `obsolete.tsv` with their replacements noted, since OBI users may have used those terms.
> 
> Can you please double-check this for me?

Looks good to me. Thanks!
